docs/source/guides/execution.md

```diff
@@ -224,19 +224,17 @@ executor = your_skypilot_cluster(nodes=8, devices=8, container_image="your-nemo-
 
 As demonstrated in the examples, defining executors in Python offers great flexibility. You can easily mix and match things like common environment variables, and the separation of tasks from executors enables you to run the same configured task on any supported executor.
 
-#### DGXCloudExecutor
+#### RunAIExecutor
 
-The `DGXCloudExecutor` integrates with a DGX Cloud cluster's Run:ai API to launch distributed jobs. It uses REST API calls to authenticate, identify the target project and cluster, and submit the job specification.
-
-> **_WARNING:_** Currently, the `DGXCloudExecutor` is only supported when launching experiments *from* a pod running on the DGX Cloud cluster itself. Furthermore, this launching pod must have access to a Persistent Volume Claim (PVC) where the experiment/job directories will be created, and this same PVC must also be configured to be mounted by the job being launched.
+The `RunAIExecutor` integrates with the Run:ai API to launch distributed jobs. It uses REST API calls to authenticate, identify the target project and cluster, and submit the job specification.
 
 Here's an example configuration:
 
 ```python
-def your_dgx_executor(nodes: int, gpus_per_node: int, container_image: str):
-    # Ensure these are set correctly for your DGX Cloud environment
+def your_runai_executor(nodes: int, gpus_per_node: int, container_image: str):
+    # Ensure these are set correctly for your RunAI environment
```

Comment:
This should technically be `NVIDIA Run:ai`. I'll add a comment to each of these I find.